src/Microsoft.DotNet.Build.Tasks.Packaging/src/GetLastStablePackage.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.Tasks.Packaging
+{
+    public class GetLastStablePackage : PackagingTask
+    {
+        /// <summary>
+        /// List of packages to look up.  Stable version less than this version will be returned.
+        ///   Identity: Package ID
+        ///   Version: Package version.
+        /// </summary>
+        [Required]
+        public ITaskItem[] LatestPackages { get; set; }
```

Comment:
It's calculated in packaging.targets based on the pkgproj being built and it's run-time dependencies.